dhis-2/dhis-services/dhis-service-tracker/src/test/java/org/hisp/dhis/tracker/export/trackedentity/TrackedEntityOperationParamsMapperTest.java

```diff
@@ -567,10 +567,10 @@ void shouldFailWhenUserHasNoAccessToAnyTrackedEntityType() {
     TrackedEntityOperationParams operationParams =
         TrackedEntityOperationParams.builder().orgUnitMode(ACCESSIBLE).build();
 
-    Exception badRequestException =
+    Exception forbiddenException =
```

Comment:
```suggestion
    Exception exception =
```

for new tests at least. The type is already in the `assertThrows` so there is less to change in a case like this.